reference/constraints/Country.rst

```diff
@@ -78,6 +79,15 @@ Options
 
 .. include:: /reference/constraints/_groups-option.rst.inc
 
+alpha3
+~~~~~~
+
+**type**: ``boolean`` **default**: ``false``
+
+If this option is ``true``, then the constraint will valid the value as
```

Comment:
I would write in the reverse way.
Default alpha2 description
And if set to true, validate against alpha3

Wdyt?

Reply:
I don't mind but I wanted to be consistent with current documentation about `alpha3` option like in country and language type in form component.